data_prep/introspector.py

```diff
@@ -42,6 +42,8 @@
 # to generate benchmarks.
 ORACLE_AVOID_STATIC_FUNCTIONS = bool(
     int(os.getenv('OSS_FUZZ_AVOID_STATIC_FUNCTIONS', '1')))
+ORACLE_ONLY_REFERENCED_FUNCTIONS = bool(
+    int(os.getenv('OSS_FUZZ_ONLY_REFERENCED_FUNCTIONS', '0')))
```

Comment:
Shall we enable this by default?

Reply:
I'd prefer this is opt-in so think we should keep it off at the moment